config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -31,22 +31,29 @@
     private static final String REGISTRY_CONF_SUFFIX = ".conf";
     private static final String ENV_SYSTEM_KEY = "SEATA_ENV";
     private static final String ENV_PROPERTY_KEY = "seataEnv";
-    /**
-     * the name of env
-     */
-    private static String envValue;
+
+    private static final String SYSTEM_PROPERTY_SEATA_CONFIG_NAME = "seata.config.name";
+
+    private static final String ENV_SEATA_CONFIG_NAME = "SEATA_CONFIG_NAME";
```

Comment:
I guess our users may feel confuse with this name as there's already a envrionment variable called SEATA_ENV.As i said above,can we use a configuration item to control the file name of registry.conf?

Reply:
I think SEATA_CONFIG_NAME does not conflict to SEATA_ENV, they have different meaning, SEATA_CONFIG_NAME is file name, and SEATA_ENV is profile name. Before I added this PR, I referred to how spring-boot custom application.properties. As spring did,  `seata.config.name` should be provided as a command line parameter, but system property is easier to implement here.